### PR TITLE
Optimization + different way how to recognize blobs for each digit

### PR DESCRIPTION
**Pull request by mcochner**
I tried to solve problem by optimizing some parameters. Also the blobs that should represent each digit (the meter has 8 blobs) are now gathered only from appropriate parts of the image. Anyway it seems that it is working better then previous version. Would be helpfull if there were some pictures that the neural network weren't trained on (test set).

### CODE REVIEW

**Comment by pavelkraleu** [comment]
Thanks for great improvements !
I will upload testing dataset shortly, I have few GB of training data which just waits for manual selection.

**Comment by pavelkraleu** [comment]
We have testing dataset. Its described in the readme https://github.com/ParalelniPolis/ppplyn#training-dataset
Detector success rate is over 99% :)
